subcommand/injector/command.go

```diff
@@ -175,6 +199,15 @@ func (c *Command) Run(args []string) int {
 	return 0
 }
 
+func getNamespace() string {
+	namespace := os.Getenv("NAMESPACE")
+	if len(namespace) > 0 {
```

Comment:
nit: `namespace != ""` seems more canonical?

Reply:
I'm good with that, updated in 501069e